### ISSUE TITLE
Accordion accessibility

### ISSUE THREAD

**Issue by maxokorokov** [ng-bootstrap/ng-bootstrap#2927]
I've been looking into WAI-ARIA recommendation:
- docs → https://www.w3.org/TR/wai-aria-practices-1.1/#accordion
- accordion example → https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html

Markup they propose for accordion is:

```html
<dl id="accordionGroup" role="presentation">
  <dt role="heading" aria-level="3">
    <button aria-expanded="true" aria-controls="sect1" id="accordion1id" type="button" aria-disabled="true">
      <span>Title 1</span>
    </button>
  </dt>
  <dd id="sect1" role="region" aria-labelledby="accordion1id">
    <div>
      <!-- Variable content within section, may include any type of markup or interactive widgets. -->
    </div>
  </dd>
</dl>
```

That has nothing to do with what we have (roles, attributes):

```html
<ngb-accordion class="accordion" role="tablist" aria-multiselectable="true">
  <div class="card">
    <div role="tab" id="ngb-panel-0-header" class="card-header ">
      <button type="button" aria-expanded="true" aria-controls="ngb-panel-0"> Simple </button>
    </div>
    <div class="collapse show" role="tabpanel" id="ngb-panel-0" aria-labelledby="ngb-panel-0-header">
      <div class="card-body">
        <!-- Content goes here -->
      </div>
    </div>
  </div>
</ngb-accordion>
```

---

Differences:
- roles: `tablist / tab / tabpanel` vs `presentation / heading / region`
- they have `aria-disabled` on a button
- they have `aria-level` on a heading
- we have `aria-multiselectable` on the accordion

Fixed:
~~- we have an intermediate`<h5>` (should be addressed by #2847)~~

**Comment by gpolychronisAmadeus** [comment]
@maxokorokov @benouat 
Please review.
Also: added ariaLevel input to ngbPanelHeader. It defaults to 3.

**Comment by IAfanasovMob** [comment]
> they have aria-disabled on a button

we have `disabled` attribute 

According to [ARIA docs](https://www.w3.org/TR/html-aria/), we do it correct
> Only use the aria-disabled attribute for elements that are not allowed to have a disabled attribute in HTML.

**Comment by IAfanasov** [comment]
I created the PR to resolve accessibility issues.
Some bullets from the @maxokorokov post outdated. I followed the current documentation and added references to ARIA documentation into ng-bootstrap docs

**Comment by radhikachitkara** [comment]
Hi Team,

We are facing similar issue in our application where 2 roles are announced for accordions 'button' and 'tab', role tab is automatically getting added to HTML.

<img width="622" alt="SS" src="https://user-images.githubusercontent.com/119498590/209281483-00832b5c-733e-4733-b476-33cac34584e0.PNG">

HTML structure:

<ngb-accordion #accordion="ngbAccordion" [activeIds]="activeId" (panelChange)="togglePanel($event)">
    <ngb-panel [id]="'p' + i" cardClass="expansion-panel">
      <ng-template ngbPanelHeader>
        <div [attr.aria-expanded]="openById['p' + i] ? true : false"
          (keyup.enter)="accordion.toggle('p' + i)" (click)="accordion.toggle('p' + i)"
          class="header-sleeve focus-visibility" [ngClass]="{ expanded: openById['p' + i] }" role="button">
          <div class="header-text d-flex justify-content-between">
            <div>Spend</div>
            <div class="m-r-20">$1100</div>
          </div>
          <div class="header-controls">
            <i *ngIf="openById['p' + i]" class="fal fa-chevron-up" aria-hidden="true"></i>
            <i *ngIf="!openById['p' + i]" class="fal fa-chevron-down" aria-hidden="true"></i>
          </div>
        </div>
        <p class="sm">Spend decs</p>
      </ng-template>
      <ng-template ngbPanelContent>
        <div class="row">
          <ng-container>
            <div class="col-8">
              <p> Plan1</p>
            </div>
            <div class="col-3 text-right">
              <p>$1100</p>
            </div>
          </ng-container>
        </div>
      </ng-template>
    </ngb-panel>
</ngb-accordion>

**Comment by maxokorokov** [comment]
This should be be fixed by #4437 with new accordion directives (#3898).
It will be a part of `14.1.0` release.

New accordion follows the bootstrap markup and https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/
